docs/docs/cli/checkpoint/delete.mdx

```diff
@@ -22,7 +22,7 @@ Checkpoints that are uploaded or generated from training jobs are physically del
 
 | Argument | Type | Summary | Default | Required | 
 |----------|------|---------|---------|----------|
-| **`checkpoint_id`** | UUID | ID of checkpoint to delete. | - | ✅ |
+| **`checkpoint_ids`** | UUID | IDs of checkpoint to delete. When multiple IDs are provided, all those checkpoints are deleted. | - | ✅ |
```

Comment:
In this command, is `checkpoint_ids` a comma-separated, or a space-separated?
I think it is better to show how to delete multiple ckpts.